portfolio/src/main/webapp/script.js

```diff
@@ -67,13 +67,14 @@ function handleResponse(response) {
   const textPromise = response.text();
 
   // When the response is converted to text, pass the result into the
-  // addMessageToDom() function.
+  // addmessageToDom() function.
   textPromise.then(addMessageToDom);
```

Comment:
If addMessageToDom were sufficiently complicated or was going to be reused somewhere else, I'd split it out into a separate named function.  But for simple functions like this, I'd just inline it with arrow notation

Reply:
will do.